### PR TITLE
Fix inputs with the autofocus attribute

### LINKED ISSUE #11892: Allow autofocus on input in amp-form
In an amp-form, an input element with the autofocus attribute does not gain focus.

See an example here:

https://codepen.io/vercer/pen/MOwdVB

I expect the first input to be focused when the page loads but it is not.  Here is an example of the expected behaviour:

https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_autofocus

### PR DESCRIPTION
**Pull request by cvializ**
Fixes #11892 

It was a quick hit so I went ahead and made a PR

### CODE REVIEW

File: extensions/amp-form/0.1/amp-form.js [hunk 1]
```diff
@@ -251,6 +257,13 @@ export class AmpForm {
 
   /** @private */
   installEventHandlers_() {
+    this.viewer_.whenFirstVisible().then(() => {
```
Comment (aghassemi) [inline extensions/amp-form/0.1/amp-form.js]: we probably want autofocus to take effect everytime document becomes visible (think viewer swipes). `whenNextVisible` does that (and covers first visibility as well)

Comment (cvializ) [reply extensions/amp-form/0.1/amp-form.js]: Ah you're right; I thought it was just once after the next time it became visible.

Comment (cvializ) [reply extensions/amp-form/0.1/amp-form.js]: Is the document state reset between viewer swipes? i.e. would the user expect the field to autofocus again if they swipe to the next document and then swipe back?

Comment (aghassemi) [reply extensions/amp-form/0.1/amp-form.js]: oh naming! :) if you can think of a shorter version of `ifAlreadyVisibleOrWhenNextVisible` , we can rename. it is a fairly new feature and not used in many places yet.

Comment (cvializ) [reply extensions/amp-form/0.1/amp-form.js]: haha yeah one of the hardest problems ; )

**Pull request merged**